draft-ietf-gnap-core-protocol.md

```diff
@@ -357,10 +355,10 @@ that returns from the interaction.
 
 ~~~
     +--------+                                  +--------+         +------+
-    | Client |                                  |   AS   |         |  RO  |
-    |Instance|                                  |        |         |  +   |
-    |        |< (1) + Start Session + + + + + + + + + + + + + + + +|  RQ  |
-    |        |                                  |        |         |(User)|
+    | Client |                                  |   AS   |         | User |
+    |Instance|                                  |        |         |      |
```

Comment:
```suggestion
    | Client |                                  |   AS   |         | End  |
    |Instance|                                  |        |         | User |
```

Reply:
Oh good catch, I missed that this one was previously RO + RQ and over-corrected.